### DISCUSSION TITLE
Add weather icon colour selection (the cyan chosen is a bit too dark), as well as outline/stroke for weather text

### DISCUSSION THREAD

**Discussion by DRSAgile** [FelixdelasPozas/TrayWeather#77] [Ideas]
Hi,

The application is great, so thanks a lot.

If possible, can weather icon colour selection be added in a future release? The cyan chosen for the icon now is a bit too dark.

Also, the option to use coloured text for the temperature would be useful more often if the text would be with an outline/stroke of an opposite/inverted colour ("^ 0xffffff" XOR operation) for contrast.

Thanks in advance.

**Comment by FelixdelasPozas** [comment]
Hi @DRSAgile .

> The application is great, so thanks a lot.

Thank you, much appreciated. 

> If possible, can weather icon colour selection be added in a future release? The cyan chosen for the icon now is a bit too dark.

I'm hesitant to add the option to modify icon colors as I never had that problem and even the simplest colored icons have many different colors. Also as the taskbar in Windows is translucent the contrast of the tray icon depends on the wallpaper used and Windows theme colors. But for the next release I will include icon themes (like [this one](https://github.com/zagortenay333/Tempestacons)). Hopefully one of them will work for you. 

> Also, the option to use coloured text for the temperature would be useful more often if the text would be with an outline/stroke of an opposite/inverted colour ("^ 0xffffff" XOR operation) for contrast.

That's a good idea, I'll add it for the next release. 

Happy holidays.
Félix.

**Comment by FelixdelasPozas** [comment]
Hi @DRSAgile 

The [latest release](https://github.com/FelixdelasPozas/TrayWeather/releases/tag/1.15.0) (version 1.15) has icon themes. You will be able to change the weather icons to a theme that suits your desktop. Also I've added an outline to the temperature text icon. 

Regards,
Félix.

**Comment by DRSAgile** [comment]
Thank you, Félix.

**Comment by adem4ik** [comment]
Text temperature outline is really great improvemen, now I can see TW icon in the Task Manager, so it fixed https://github.com/FelixdelasPozas/TrayWeather/issues/22 . Many thanks!

![image](https://user-images.githubusercontent.com/4707112/147866556-e0cf0129-4056-47ab-9036-48061a577580.png)
